polkadot/runtime/parachains/src/hrmp.rs

```diff
@@ -270,6 +274,10 @@ pub mod pallet {
 		/// implementation should be the same as `Balance` as used in the `Configuration`.
 		type Currency: ReservableCurrency<Self::AccountId>;
 
+		/// The default channel size and capacity to use when opening a channel to a system
+		/// parachain.
```

Comment:
Does it need metadata with `pallet::constant`?

Reply:
we can do it. but given the max value are not constant, do we want to make those constant? maybe we want to use the parameter pallet to specify the value?

Reply:
Ah yea, i am often not sure what to put in metadata 😅